### DISCUSSION TITLE
Problem running IMAGE apache/airflow:2.2.0.dev0 in docker

### DISCUSSION THREAD

**Discussion by Adefioye** [apache/airflow#17893] [Q&A] [answered]
`(base) PS C:\Users\abdul\airflow-test> **docker-compose up airflow-init**
Creating network "airflow-test_default" with the default driver
Creating volume "airflow-test_postgres-db-volume" with default driver
Pulling airflow-init (apache/airflow:2.2.0.dev0)...
**ERROR: manifest for apache/airflow:2.2.0.dev0 not found: manifest unknown: manifest unknown**
(base) PS C:\Users\abdul\airflow-test> **docker-compose up airflow-init**
Pulling airflow-init (apache/airflow:2.2.0.dev0)...
**ERROR: manifest for apache/airflow:2.2.0.dev0 not found: manifest unknown: manifest unknown**

I ran docker-compose run airflow-init` based on instructions on the link  **http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/start/docker.html** and I have been getting **manifest unknown** error.

Based on my research, I realise this is because the image is not on docker repository. I tried using **apache/airflow:2.1.3** and I got an error that the image version is old and that I should use **2.2.0**. Upon using it, I am having same manifest unknown error.

### I am using docker desktop on windows

**Comment by potiuk** [comment]
This is the problem in our docs. I will fix it now.

**Reply by potiuk** [reply]
Update. There were some 2.2.0.dev0 references in released docs (now fixed by https://github.com/apache/airflow-site/pull/473) but the root cause was different - dev documentation from s3 was used and not the official doc.

**Comment by potiuk** [comment]
Fix here: https://github.com/apache/airflow-site/pull/473

**Comment by potiuk** [comment, marked as answer]
Actually. The original report was kind of wrong as well.  The http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/start/docker.html is develoment version build from main, so there you will find "dev" links whcih won't work. You should use https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html which is about released version

**Reply by Adefioye** [reply]
Hi! Thanks just ran the docker-compose file now. It works fine
